src/lightly_train/_models/package_helpers.py

```diff
@@ -58,32 +62,28 @@ def list_model_names() -> list[str]:
 
     See the documentation for more information: https://docs.lightly.ai/train/stable/models/
     """
-    return sorted(chain.from_iterable(p.list_model_names() for p in list_packages()))
-
-
-def get_model(model: str | Module, model_args: dict[str, Any] | None = None) -> Module:
-    """Returns a model instance given a model name or instance."""
-    if isinstance(model, Module):
-        return model
-
-    package_name, model_name = _parse_model_name(model=model)
-    package = get_package(package_name=package_name)
-    return package.get_model(model_name, model_args)
-
+    return sorted(
+        chain.from_iterable(p.list_model_names() for p in list_full_packages())
+    )
 
-def get_model_wrapper(model: Module) -> ModelWrapper:
-    """Returns a model wrapper class for the given model."""
-    for package in list_packages():
-        if package.is_supported_model(model):
-            return package.get_model_wrapper(model)
 
-    raise UnknownModelError(f"Unknown model: '{model.__class__.__name__}'")
+def get_wrapped_model(
+    model: str, model_args: dict[str, Any] | None = None
+) -> ModelWrapper:
+    """Returns a wrapped model instance given a model name or instance."""
+    package_name, model_name = _parse_model_name(model)
+    package = get_package(package_name)
+    # This can never be a BasePackage, because calling get_wrapped_model() makes no sense
+    # for a BasePackage.
```

Comment:
```suggestion
    package = get_package(package_name)
    # This can never be a BasePackage, because calling get_wrapped_model() makes no sense
    # for a BasePackage.
```

This looks to me like `get_package` must have return type `Package` instead of `BasePackage`.

Reply:
True, thanks for the catch.